config/locales/hyrax.en.yml

```diff
@@ -1,10 +1,10 @@
 ---
 en:
   hyrax:
-    product_name:           "Hyku"
+    product_name:           "Hyku Commons"
```

Comment:
change this back to "Hyku" for this and all other languages

Reply:
Changed back to "Hyku" for all languages.